Added support for nxtgn.org

Inrego opened the pull request:
I added support for a danish tracker at nxtgn.org

Commit:
Added support for nxtgn.org

Kayomani commented on src/Jackett/Indexers/NxtGn.cs:
```diff
@@ -0,0 +1,185 @@
+            var releases = new List<ReleaseInfo>();
+            var breakWhile = false;
+            var page = 0;
+            while (true)
```
Can you make this a for loop any only allow three pages? A little concerned you could search for 2015 etc and it'll hit the server who knows how many times?

Kayomani commented on src/Jackett/Indexers/NxtGn.cs:
```diff
@@ -0,0 +1,185 @@
+                        release.Peers = ParseUtil.CoerceInt(qRow.Find("#torrent-leechers").Text().Trim()) + release.Seeders;
+
+                        var sizeStr = qRow.Find("#torrent-size").First().Text();
+                        release.Size = ReleaseInfo.GetBytes(sizeStr);
```
The mapping of the release to a category is missing:

 release.Category = MapTrackerCatToNewznab(cat);

I see you have several mappings to a a single category so you might need to do something similar to that in pretome where you map the output to tv rather than tv/sd or tv/hd.

Inrego replied:
I don't quite understand what you mean. I don't see any mappings in pretome. Could you elaborate please? I can't see category being set anywhere in pretome

Kayomani replied:
Do a pull :)

ghost commented:
Couple of minor changes but otherwise looks good  :+1: 

Could you also make the change to develop rather than master please?

ghost commented:
I am doing a release in a couple of hours if you want to aim for that cut.

Kayomani commented on src/Jackett/Jackett.csproj:
```diff
@@ -1,594 +1,596 @@
+    <Content Include="Content\logos\ncore.png">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Content\logos\nxtgn.png" />
```
Needs  <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>

Commit:
Will now only look at the first 3 pages when searching. To prevent flooding server when searching for a common term.

A category is now assigned to releases.

Changed nxtgn banner to only be copied if newer upon build.

Inrego commented:
I'm a bit new to git. You asked me to push the changes to develop aswell. To do that, would I need to fork the develop branch and add the same changes there and then do a pull request?

Also, I believe I made the changes you requested. Is it too late for the release?

ghost commented:
git checkout develop (switch to develop)
git merge upstream/develop (Get our latest changes)
git rebase master (Copy changes from master to develop)

https://help.github.com/articles/syncing-a-fork/

On 23 August 2015 at 23:41, Inrego notifications@github.com wrote:

> I'm a bit new to git. You asked me to push the changes to develop aswell.
> To do that, would I need to fork the develop branch and add the same
> changes there and then do a pull request?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zone117x/Jackett/pull/146#issuecomment-133950341.

ghost commented:
Then push the change your develop then make a pull request from there ;D

ghost commented:
Let me know if you are having any issues.

Inrego commented:
Yeah sorry, I'm busy with work. I'll get to it during the week.

Commit:
Added imdb ID for movies.
